report/sections/Conclusion/conclusion.tex

```diff
@@ -0,0 +1,21 @@
+The main goal of this project was to create an internet-based application that would solve some problem.
+Furthermore, another requirement in the study regulation is that the developed system also needs to be tested for scalability.
+In this project, we developed a system for lecturers of the \textit{Programming Paradigms} course that can provide fast and efficient feedback on student exercise submissions.
+The system allows lecturers to create exercises for students and define accompanying tests for verifying correctness of student solutions.
+
+To create this application, we developed a working user interface that students and lecturers can use in order to interact with the system.
```

Comment:
```suggestion
To create this application, we developed a working user interface that students and lecturers can use in order to interact with the system. The user interface and an accompanying server for handling interactions was developed using TypeScript, tRPC and Next.js.
To enable persistance of users and their interactions with the system, a login system was implemented using a layered architecture.
Information regarding the users, their submitted exercise solutions, and exercises and accompagnying tests was persisted on a Postgres database.
```

Reply:
Read below review comment before reviewing.

Reply:
@TheRumle You have spelling errors. `accompagnying` should be accompanying.
And you missed Prisma, which would be great to mention as we use it to communicate with the database.

```suggestion
To create this application, we developed a user interface that students and lecturers can use in order to interact with the system.
```
`working` should be given, otherwise we could have just handed in mockups.